web/mobile.html

```diff
@@ -10,7 +10,7 @@
     <meta http-equiv="refresh" content="0;URL='https://dartpad.dartlang.org/index.html'" />    
   </head>    
   <body> 
-    <p>This page has moved to a <a href="https://dartpad.dartlang.org/index.html">
-      https://dartpad.dartlang.org/</a>.</p> 
+    <p>This page has moved to a <a href="https://dartpad.dev/index.html">
```

Comment:
I'd probably make this link to https://dartpad.dev/ instead. =)

Reply:
Should be good now :)